src/rt/util/hash.d

```diff
@@ -95,15 +90,18 @@ size_t hashOf( const(void)* buf, size_t len, size_t seed )
     return hash;
 }
 
-// Check that hashOf works with CTFE
-@nogc nothrow pure unittest
+unittest
 {
-    size_t ctfeHash(string x)
-    {
-        return hashOf(x.ptr, x.length, 0);
-    }
-
     enum test_str = "Sample string";
-    enum size_t hashVal = ctfeHash(test_str);
-    assert(hashVal == hashOf(test_str.ptr, test_str.length, 0));
+    size_t hashval = hashOf(test_str, 5);
```

Comment:
> Can we postpone this PR

We'll never make any progress doing things like that. CTFE is irrelevant to hashOf, there are no uses of it. One thing at a time.